settings.gradle.kts

```diff
@@ -93,6 +93,7 @@ include(":subprojects:android-test:toast-rule")
 include(":subprojects:android-lib:proxy-toast")
 include(":subprojects:android-test:test-app-kaspresso")
 include(":subprojects:android-test:test-instrumentation-runner")
+include(":subprojects:android-test:test-app-1")
```

Comment:
Let's call it more specifically.
`test-app-demo`
`test-app-extra`
...

Anything must be more understandable than `1`

Reply:
Agree, the main proposition is creating a sample for issue in Gradle